core/src/player.rs

```diff
@@ -677,6 +677,17 @@ impl Player {
         );
     }
 
+    pub fn run_context_menu_fullscreen_callback(&mut self, is_fullscreen: bool) {
```

Comment:
This should probably be called `set_is_fullscreen`, or `did_enter_fullscreen`, or something. This name is a bit too verbose and less descriptive as to what it does.